hack/test-cmd.sh

```diff
@@ -148,15 +128,11 @@ openshift start \
   --images="${USE_IMAGES}"
 
 
-# Don't try this at home.  We don't have flags for setting etcd ports in the config, but we want deconflicted ones.  Use sed to replace defaults in a completely unsafe way
-sed -i "s/:4001$/:${ETCD_PORT}/g" ${SERVER_CONFIG_DIR}/master/master-config.yaml
```

Comment:
This `sed` operation is pretty basic.  How do you express a global find and replace on mac os?  According to this https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/sed.1.html `-i` is supported and so is `s//g`.